### ISSUE TITLE
Do not report duplicated content on the first occurence

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#482]
In case of our duplication rules we're reporting issues on every duplicated line (including original). Modify it so it doesn't report on original. Mention original line in the message:
```
test.robot:13:1 [E] 0801 Multiple test cases with name "Test" (first occurrence in line 6)
```
